srtcore/srt.h

```diff
@@ -561,6 +561,14 @@ enum SRT_REJECT_REASON
     SRT_REJ__SIZE,
 };
 
+// Reject category codes:
```

Comment:
There's a proposed idea for the `SRT_REJECT_REASON` enum to mark pegs with numbers every 5 values, for clarity.

Reply:
```
    SRT_REJ_UNKNOWN = 0,     // initial set when in progress
    SRT_REJ_SYSTEM,      // broken due to system function error
    SRT_REJ_PEER,        // connection was rejected by peer
    SRT_REJ_RESOURCE,    // internal problem with resource allocation
    SRT_REJ_ROGUE,       // incorrect data in handshake messages
    SRT_REJ_BACKLOG = 5,     // listener's backlog exceeded
    SRT_REJ_IPE,         // internal program error
...
```

Reply:
I still don't understand what is proposed? Marking every 5th element? How does it belong to this comment line?
Ah, you probably meant to address the whole enum above?

Reply:
Yes. It came out in one of the talks that when the marking numbers are too distant to particular label, it's not easy to map the number to label when interpreting the logs.